### PR TITLE
Update elixir from 1.5.1 to 1.5.2

### PR DESCRIPTION
**Pull request by cstrouse**
This is a general bugfix and maintenance release.

Tested as working on XE500C13-K01US.

### CODE REVIEW

File: packages/elixir.rb [hunk 1]
```diff
@@ -28,10 +20,10 @@ def self.build
...
   def self.install
-    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}"
-    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/share"
-    system "mv bin #{CREW_DEST_DIR}#{CREW_PREFIX}"
-    system "mv lib #{CREW_DEST_DIR}#{CREW_PREFIX}"
-    system "mv man #{CREW_DEST_DIR}#{CREW_PREFIX}/share"
+    system "mkdir -p #{CREW_DEST_PREFIX}"
+    system "mkdir -p #{CREW_DEST_PREFIX}/share"
+    system "mv bin #{CREW_DEST_PREFIX}"
+    system "mv lib #{CREW_DEST_PREFIX}"
```
Comment (uberhacker) [inline packages/elixir.rb]: Should this line be `system "mv lib/* #{CREW_DEST_LIB_PREFIX}"` ?

**Comment by uberhacker** [comment]
@cstrouse: Did you get a chance to look into the lib directory abstraction yet?

**Comment by cstrouse** [comment]
Sorry @uberhacker I've been using a hotspot and these larger packages are slow and tedious to work with sometimes when the network is slow. It probably should be changed to use install and set guarantee appropriate permissions; that is just how it was and appeared to be working.

**Comment by uberhacker** [comment]
Does it work on x86_64 if you take out `/usr/local/lib` from the library search path?  That should be your indicator.

**Comment by cstrouse** [comment]
I've modified the package to use CREW_DEST_LIB_PREFIX. Works properly on ARMv8.

File: packages/elixir.rb [hunk 2]
```diff
@@ -3,35 +3,28 @@
...
   binary_sha256 ({
-    aarch64: '54a5afb5f1a6ac89dde31da18c8d8c0982de24a8f6235fb6dc680bf4faeb1962',
-     armv7l: '54a5afb5f1a6ac89dde31da18c8d8c0982de24a8f6235fb6dc680bf4faeb1962',
-       i686: 'a9e86ddd0120238198f7aea64b148818034b7d9a75b7f56f8c1b79c67bae5098',
-     x86_64: '01a4e221a51f9c73ab55906c1d1b9676435a551743482c72c16200ad62c4648e',
   })
 
   depends_on 'erlang'
-  depends_on 'unzip'
+  depends_on 'unzip' => :build
```
Comment (uberhacker) [inline packages/elixir.rb]: I'm not noticing anything in the build that would suggest we need `unzip`.  It doesn't look like the install section uses it either.

**Pull request merged**